doc/index.md

```diff
@@ -0,0 +1,521 @@
+
+DSA apps support several generic message feeds.
+
+| | |
+| --- | --- |
+| SA Events |  Situational Awareness (SA) events, using the Cursor on Target (CoT) format. |
+| Friendly Tracks – Land |  Position Reports from land-based friendly team members. Track information is symbolized using a MIL-STD-2525C symbol ID code and draped on the elevation surface. |
+| Friendly Tracks – Air | Position reports from friendly airborne team members. Track information is symbolized using a MIL-STD-2525C symbol ID code and are displayed using a z-value as an absolute distance above the elevation surface. |
+| Observation Reports | Observations made while in the field, using a typical "SALUTE" format (Size, Activity, Location, Unit ID, Time, Equipment). |
+| Situation Reports | View teammates' current situation. |
+| EOD Reports | Explosive Ordinance Disposal (EOD) mission reports. |
+| Sensor Observations | Observations made by sensors in the field. |
+
+***Developer tips:***
+
+- DSA serializes feeds as XML, which is then converted into bytes. Next, the bytes are broadcast as datagrams over a specific UDP port. DSA apps are configured to listen on the same UDP ports, so when incoming datagrams are received, the messages are deserialized and displayed on the map.
+- This app uses [dynamic rendering] for graphics.
+- Military symbols are displayed using a [dictionary renderer].
+
+## Exploratory visual analysis
+
+Exploratory [analysis tools] combine information such as feature location, your location, and elevation to answer questions about the current operational environment. Analysis is continuously updated as the environment changes, leveraging the GPU of the device and the data that is drawn on the map. The results of an analysis are presented in an overlay that can be [managed like other overlays](#overlays).
+
+### ![](./images/dsa-icon-viewshed-32.png) Viewshed
+
+![](./images/dsa-tool-viewshed.png "Viewshed")<br>*Viewshed analysis (DSA Vehicle)*
+
+[Viewshed analysis] is a type of visibility analysis that shows the visible and obstructed areas within a directed field of view. Viewshed analysis shows you the areas of the scene that are visible from a given observer, determined by the terrain (represented by an elevation surface), buildings and other 3D features (represented by scene layers, graphics, and so on), and the properties of the observer. A viewshed is adjustable to show what is visible within a specified distance in a certain direction or all directions. Areas that are visible from the location are highlighted in green, and not-visible areas are highlighted in red. As the location moves, the viewshed analysis is recalculated.
+
+<!-- TODO - Add details -->
```

Comment:
This line can probably be removed. Likely accidentally left in during the original authoring.